docs/map-making/how-to-upgrade-legacy-map-features.md

```diff
@@ -35,70 +19,36 @@ All occurrences of the code before the `>>` must be substituted with the code af
 - `="isTwoHit" count="-reset-false"` >> `="hitPoints" count="-reset-1"`
 - `="occupiedTerrOf"` >> `="originalOwner"`
 - `="victoryCity" count="true"` >> `="victoryCity" count="1"`
-- `="victoryCity" count="-reset-true"` >> `="victoryCity" count="-reset-1"`
 - `="victoryCity" count="false"` >> `="victoryCity" count="0"`
+- `="victoryCity" value="true"` >> `="victoryCity" count="1"`
+- `="victoryCity" value="false"` >> `="victoryCity" count="0"`
+- `="victoryCity" count="-reset-true"` >> `="victoryCity" count="-reset-1"`
 - `="victoryCity" count="-reset-false"` >> `="victoryCity" count="-reset-0"`
-
-
-### **Procedural alternatives:**
-
-For a more defined change, instead of the aforementioned:
-
-- `attatchment` >> `attachment`
-
-you can change:
-
-- `attatchmentList` >> `attachmentList`
-- `<attatchment` >> `<attachment`
-- `attatchment>` >> `attachment>`
-- `="games.strategy.triplea.attatchments` >> `="games.strategy.triplea.attachments`
-
-- `="techAttatchment` >> `="techAttachment`
-- `="unitAttatchment` >> `="unitAttachment`
-- `="territoryAttatchment` >> `="territoryAttachment`
-- `="canalAttatchment` >> `="canalAttachment`
-- `="rulesAttatchment` >> `="rulesAttachment`
-- `="playerAttatchment` >> `="playerAttachment`
+- `<option name="isParatroop"` >> `<option name="isAirTransportable"`
+- `<option name="isMechanized"` >> `<optiona name="isLandTransportable"`
 
 
 ### XML Deletions
 
-_(these items were already deprecated and did nothing)_
-
-- `<option name="isParatroop"*/>`
-- `<option name="isMechanized"*/>`
-- `<option name="takeUnitControl"*/>`
-- `<option name="giveUnitControl" value="true"/>`
-- `<option name="giveUnitControl" value="false"/>`
+Delete these options entirely:
 
-## map.properties Changes
+- `<option name="takeUnitControl"`
+- `<option name="giveUnitControl"`
 
-Within every "map" properties file, inside the main folder of the map, change:
+## map.properties changes
 
 - `color.Impassible=` >> `color.Impassable=`
 
-## Sound file changes
-
-Change all sounds inside the "sounds" main folder of the map from "wav" to "mp3" format.
-128Kbps and 168Kbps are the preferred bit rates.
-
-
-## Warnings
-
-The following xml codes, as being properties, are not validated (if you leave them, the game will play,
-but they will set nothing in the engine) (it is suggested you double check you don't have them within
-any game files, after upgrading the map):
+## sound.properties and sound file changes
 
-- `="Battleships repair at end of round"`
-- `="Battleships repair at beginning of round"`
-- `="Naval Bombard Casualties Return Fire Restricted"`
-- `="SBR Affects Unit Production"`
+- .wav files need to be re-encoded as mp3 (128 or 168Kbps are preferred bit rates)
+- all file names in sound.properties need to be updated to match the new '.mp3' suffix
+  on sound files.
 
-While all options named "takeUnitControl" must be deleted, any option named "giveUnitControl" must be
```

Comment:
To limit the risk of mistakenly remove a good "giveUnitControl" option, I think this warning (comprising the next line) is valuable even if the referenced item would be correct after this update.

Reply:
I don't think this reduces the risk of mistakenly removing a valid option. If someone follows this guide exactly, then it won't be an issue, right? But furthermore, this is becoming a pure change log for information purposes, not to be followed with find/replacements. Specifically if we can load 1.8 maps in 2.3, then the right process will be to load the map and re-export it.